src/app/components/search/SearchFields/index.js

```diff
@@ -163,91 +165,92 @@ const SearchFields = ({
   */
   const updateStateQueryArrayValue = (key, newArray) => {
     if (newArray === undefined) {
-      const newQuery = { ...query };
+      const newQuery = { ...stateQuery };
       delete newQuery[key];
       return newQuery;
     }
-    return { ...query, [key]: newArray };
+    return { ...stateQuery, [key]: newArray };
   };
 
   const handleAddField = (field) => {
-    const newQuery = { ...query };
+    const newQuery = { ...stateQuery };
 
     if (field === 'team_tasks') {
-      newQuery.team_tasks = query.team_tasks ?
-        [...query.team_tasks, {}] : [{}];
+      newQuery.team_tasks = stateQuery.team_tasks ?
+        [...stateQuery.team_tasks, {}] : [{}];
     } else if (field === 'range') {
       newQuery.range = { created_at: {} };
     } else {
       newQuery[field] = [];
     }
 
-    setQuery(newQuery);
+    setStateQuery(newQuery);
   };
 
   const handleRemoveField = (field) => {
-    const newQuery = { ...query };
+    const newQuery = { ...stateQuery };
     delete newQuery[field];
-    setQuery(newQuery);
+    setStateQuery(newQuery);
   };
 
-  const filterIsActive = () => Object.keys(query).filter(k => k !== 'keyword').length > 0;
+  // const filterIsActive = () => Object.keys(query).filter(k => k !== 'keyword').length > 0;
```

Comment:
As per our pair-review, this line needs to be deleted.

Reply:
Done